examples/jsm/renderers/webgpu/utils/WebGPUBindingUtils.js

```diff
@@ -204,6 +204,27 @@ class WebGPUBindingUtils {
 
 				entriesGPU.push( { binding: bindingPoint, resource: textureGPU.sampler } );
 
+			} else if ( binding.isSampledTexture && binding.texture.isVideoTexture ) {
+
+				const textureData = backend.get( binding.texture );
+
+				let resourceGPU;
+
+				if ( binding.texture.source !== undefined && binding.texture.source.data !== undefined ) {
+
+					textureData.externalTexture = binding.texture.source.data;
```

Comment:
What's the difference in all this code compared to the `binding.isSampledTexture` condition except this one line and `binding.texture.source !== undefined && binding.texture.source.data !== undefined` part?

I think it could be moved into the next block instead.

Reply:
Also I get the same error as Sunag.